### ISSUE TITLE
Binning should be <= not <

### ISSUE THREAD

**Issue by Jwink3101** [VandyChris/Global-Sensitivity-Analysis#4]
In the python version of `MGSA_FirstOrder`, I think the line `sub = cdf_input_i < U_i[j+1]` (100) should be `sub = cdf_input_i <= U_i[j+1]`. Essentially, you never count the last value of `Output`. You can see this by noting that at the end of the inner `j` loop (104) `output_i` is not empty.

**Comment by Jwink3101** [comment]
I have not checked the matlab versions

**Comment by kdneal** [comment]
I agree with the above comment, and it applies to the MATLAB version (MGSA_FirstOrder.m) also. For large number of samples this has very small impact on Sobol' estimate, but may be more pronounced when fewer samples are used. It just leaves off the sample(s) for which the empirical CDF=1.
